packages/actor-query-operation-construct/lib/ActorQueryOperationConstruct.ts

```diff
@@ -45,10 +46,19 @@ export class ActorQueryOperationConstruct extends ActorQueryOperationTypedMediat
       await this.mediatorQueryOperation.mediate({ operation, context }),
     );
 
+    // Check if the query if it's a DESCRIBE query
+    let isDescribeOperator = false;
+    const describeQuery = context.get(new ActionContextKey('@comunica/action-query-operation:root'));
```

Comment:
Instead of defining a new context key here, can we add a constant one in https://github.com/comunica/comunica/blob/master/packages/context-entries/lib/Keys.ts ?

As name, I would suggest `@comunica/actor-query-operation-construct:localizeBlankNodes`, which just contains a boolean (that is set to `false` by the describe actor).

That way, we make the construct actor more loosely coupled to the describe actor.

Reply:
> Instead of defining a new context key here, can we add a constant one in https://github.com/comunica/comunica/blob/master/packages/context-entries/lib/Keys.ts ?
> 
> As name, I would suggest `@comunica/actor-query-operation-construct:localizeBlankNodes`, which just contains a boolean (that is set to `false` by the describe actor).
> 
> That way, we make the construct actor more loosely coupled to the describe actor.

if we want to add the key as a constant we might want to name it `@comunica/actor-query-operation:localizeBlankNodes` to make the name more general?

Reply:
Yes, good suggestion, that's a bit more future-proof indeed!